tests/unit/parser.js

```diff
@@ -6001,6 +6001,20 @@ exports["test for line breaks with 'yield'"] = function (test) {
     .addError(13, "Missing semicolon.");
 
   run.test(code, {moz: true});
+
+  var code2 = [
+    "function* gen() {",
+    "  yield",
```

Comment:
I think its a good example of why you need to know what your doing if you don't add semi-colons.